src/components/common/customerService/CustomerServiceHeader.styled.ts

```
@@ -0,0 +1,44 @@
+import { Link } from 'react-router-dom';
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)